metamer/ftest/src/test/java/org/richfaces/tests/metamer/ftest/extension/arquillian/JBossPatchInstaller.java

```diff
@@ -70,63 +50,24 @@ private static boolean isUsingJBossContainer() {
     /**
      * Apply patch from given path.
      */
-    private void applyPatch(String pathToPatch) throws CommandLineException {
+    private void applyPatch(String pathToPatch) {
```

Comment:
Why there is used `System.exit`? Wouldn't it make sense to throw exception? System.exit called in different method than main method doesn't seem right.